superset-frontend/src/views/CRUD/data/dataset/AddDataset/Footer/index.tsx

```diff
@@ -17,7 +17,65 @@
  * under the License.
  */
 import React from 'react';
+import Button from 'src/components/Button';
+import { t } from '@superset-ui/core';
+import { useSingleViewResource } from 'src/views/CRUD/hooks';
+import withToasts from 'src/components/MessageToasts/withToasts';
+import { DatasetObject } from '../types';
 
-export default function Footer() {
-  return <div>Footer</div>;
+interface FooterObject {
```

Comment:
Could we rename this to be `FooterProps`? So it's more consistent with how we name our prop interfaces in other components.

Reply:
Sure!

Reply:
+1